installing/installing_vsphere/upi/upi-vsphere-preparing-to-install.adoc

```diff
@@ -0,0 +1,35 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="upi-vsphere-preparing-to-install"]
+= Preparing to install a cluster using user-provisioned infrastructure
+include::_attributes/common-attributes.adoc[]
+:context: upi-vsphere-preparing-to-install
+
+toc::[]
+
+You prepare to install an {product-title} on vSphere by:
```

Comment:
Also, per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording) we should avoid introducing lists with incomplete sentences, and instead use full sentences (seems like it's a translation thing):
```suggestion
You prepare to install an {product-title} on vSphere by performing the following actions:
```

However you'd prefer to complete the sentence is up to you, the suggestion is just one way to do it.